### DISCUSSION TITLE
Check bot blocked exception

### DISCUSSION THREAD

**Discussion by ranjitsingha** [eternnoir/pyTelegramBotAPI#2268] [General]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using? Ans: Latest

2. What OS are you using? Ans : Latest

3. What version of python are you using? Ans : 3.10

Is there any exception method by which i can check the bot was blocked by user.
Example: except BotBlockedException as e:

**Comment by coder2020official** [comment]
Use the description of the exception to distinguish between blocked user and other errors
